md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java

```diff
@@ -20,12 +20,13 @@
 import java.io.OutputStream;
 import java.io.PrintWriter;
 import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Created by arijit on 12/8/14.
  */
 public class BaseStructure {
-    private static Logger LOGGER = Logger.getLogger(BaseStructure.class);
+    private static final Logger LOGGER = Logger.getLogger(BaseStructure.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)